code/controllers/subsystem/job.dm

```diff
@@ -434,7 +434,9 @@ SUBSYSTEM_DEF(job)
 			to_chat(M, "<b>You are playing a job that is important for Game Progression. If you have to disconnect, please notify the admins via adminhelp.</b>")
 		if(CONFIG_GET(number/minimal_access_threshold))
 			to_chat(M, "<FONT color='blue'><B>As this station was initially staffed with a [CONFIG_GET(flag/jobs_have_minimal_access) ? "full crew, only your job's necessities" : "skeleton crew, additional access may"] have been added to your ID card.</B></font>")
-
+	if(ishuman(H))
+		var/mob/living/carbon/human/wageslave = H
+		to_chat(M, "<b><span class = 'big'>Your account ID is [wageslave.account_id]. Write this down somewhere.</span></b>")
```

Comment:
can this be added to wageslave memories as well as being put in chat

Reply:
Yeah that should probably go into their memory.

Reply:
thank you for the support

Reply:
will do after the next testmerge

Reply:
should force them to add it on their own so if they forget they have to contact our new department, tech support